docs/src/main/sphinx/develop/table-functions.rst

```diff
@@ -77,7 +77,7 @@ invocation:
 .. code-block:: java
 
     ScalarArgumentSpecification.builder()
-            .name("row_count")
+            .name("ROW_COUNT")
```

Comment:
Subsequent example of `TableFunctionAnalysis analyze()` still use lowercase. Could you update them as well? 

```java
        long columnCount = (long) ((ScalarArgument) arguments.get("column_count")).getValue();
        long rowCount = (long) ((ScalarArgument) arguments.get("row_count")).getValue();
```

Reply:
Done! Thanks for catching that. I left them lowercase in error messages though. That should look better if they're case insensitive.